src/legacy/core_plugins/kibana/public/context/api/context.js

```diff
@@ -54,110 +61,93 @@ function fetchContextProvider(indexPatterns, Private) {
   const SearchSource = Private(SearchSourceProvider);
 
   return {
-    fetchPredecessors,
-    fetchSuccessors,
+    // @ts-ignore / for testing
+    fetchPredecessors: (...args) => fetchSurroundingDocs('predecessors', ...args),
```

Comment:
Is there a specific reason to keep them? If it's just to have a nice function name in the tests, I would move this line over there to keep the API as clean as possible.

Reply:
in my ongoing typescript version, I've removed them:
https://github.com/kertal/kibana/tree/pr-31424-date_nanos-context-typescript